datafusion/core/src/physical_plan/joins/nested_loop_join.rs

```diff
@@ -352,24 +337,22 @@ fn build_join_indices(
 }
 
 impl NestedLoopJoinStream {
-    fn poll_next_impl(
+    // For Inner/Left/LeftSemi/LeftAnti joins, left is the single partition side.
```

Comment:
maybe we should add `full` join type.

Reply:
The meaning of the comment is opposite :rofl:.

When left is collected to a single partition, the join type should be `Right/RightSemi/RightAnti/Full`.

Fixed it.

Reply:
> The meaning of the comment is opposite 🤣.
> 

yes, I also ignore this and just remember the missing type of `Full`.

Thanks for your remind and point out the mistake of the comments.

👍

> When left is collected to a single partition, the join type should be `Right/RightSemi/RightAnti/Full`.
> 
> Fixed it.